include/GafferUI/ViewportGadget.h

```diff
@@ -103,6 +103,11 @@ class GAFFERUI_API ViewportGadget : public Gadget
 		bool getCameraEditable() const;
 		void setCameraEditable( bool editable );
 
+		/// The centre of interest is the depth (in camera space)
+		/// of a pivot about which the Alt+drag camera motion operates.
+		void setCentreOfInterest( float centreOfInterest );
```

Comment:
I know I'm nitpicking in favor of my own dialect, but should we perhaps call this `setCenterOfInterest` to match the spelling in [Grid](https://github.com/GafferHQ/gaffer/blob/e05c68138af6049a66f3bcf8c85adb15e3313af6/include/GafferScene/Grid.h#L79), [Crop](https://github.com/GafferHQ/gaffer/blob/e05c68138af6049a66f3bcf8c85adb15e3313af6/include/GafferImage/Crop.h#L76), [Constraint](https://github.com/GafferHQ/gaffer/blob/e05c68138af6049a66f3bcf8c85adb15e3313af6/include/GafferScene/Constraint.h#L67), etc.

Reply:
Done in 7c50e79ae06d3ad795e16f0a7d2d8f774491985e. There's a few more spots that we could bring into line if you want to take a look at that at some point.